Tests/Common/Utils/TestRng.swift

```diff
@@ -38,4 +38,17 @@ func testRngCallback(context: UnsafeMutableRawPointer!) -> UInt64 {
     context.assumingMemoryBound(to: Xoshiro.self).pointee.next()
```

Comment:
```suggestion
    context.assumingMemoryBound(to: MobileCoinXoshiroRng.self).pointee.nextUInt64()
```

Reply:
this fixes the error I posted in that screenshot

Reply:
Thanks -